cms/templates/partials/admissions-section.html

```diff
@@ -0,0 +1,43 @@
+{% load image_version_url wagtailcore_tags %}
+<div class="container-fluid mb-4 admissions-section">
+    <div class="row">
+        <div class="container admissions-section">
+            <div class="row">
+                <div class="col-md-5 col-sm-12 mb-2">
+                    <h1 class="section-heading">Information</h1>
+                    <h2 class="admissions-heading mt-2">Admissions</h2>
+                    <a href="/admissions" class="btn btn-primary btn-apply mt-4 mb-2">Apply Now</a>
+                    {% if page.notes %}
+                        <br />
+                        <small class="ml-3">*{{ page.notes }}</small>
+                    {% endif %}
+                </div>
+                <div class="col-md-7 col-sm-12 border-left border-dark">
+                    {{ page.details|richtext }}
+                    <img src="{% image_version_url page.workflow_infographics "max-600x400" %}" width="100%" height="auto" alt="bootcamp steps">
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row py-5 very-light-grey">
+        <div class="container">
+            <div class="row text-med-dark-grey">
+                <div class="col-md-4 col-sm-12 border-left border-dark">
+                    <h3 class="block-title">Hosted By</h3>
+                    <h2 class="font-weight-bold pb-2">{{ page.hosting_organization }}</h2>
+                    <h5 class="font-weight-bold">{{ page.hosting_location }}</h5>
+                </div>
+                <div class="col-md-4 col-sm-12 border-left border-dark">
+                    <h3 class="block-title">Bootcamp Dates</h3>
+                    <h2 class="font-weight-bold pb-2">{{ page.dates }}</h2>
+                    <h5 class="font-weight-bold">{{ page.duration }}</h5>
+                </div>
+                <div class="col-md-4 col-sm-12 border-left border-dark">
+                    <h3 class="block-title">Tuition</h3>
+                    <h2 class="font-weight-bold pb-2">{{ page.price }}</h2>
```

Comment:
`$` sign should be before price value

Reply:
also `USD` after price.

Reply:
AC details text field, so no markers or currency codes.

Reply:
there should be `$` sign before price.

Reply:
Price field is a **text** field, so the "$" can come in there, otherwise we would have had it as a decimal field. AC clearly specifies we need to keep it simple for now and make most things text fields.

Reply:
Actually, when It is asking me for Price then I am thinking to put a value `100` instead of `$100 USD`.

Reply:
It's a text field as per specs, not an integer or float field which only has value.